src/pages/AddEmployee.jsx

```diff
@@ -141,19 +160,21 @@ function AddEmployee({ open, onClose }) {
                     color="success"
                     disabled={isPending}
                   >
-                    Add
+                    Save
                   </Button>
                 </Grid>
               </Grid>
             </form>
           </CardContent>
         </Card>
-        <Toast
-          open={openToast}
-          onClose={() => setOpenToast(false)}
-          message={toastMessage}
-          severity={toastSeverity}
-        />
+        {toast.open && (
+          <Toast
+            onClose={() => setToast({ ...(open = false) })}
```

Comment:
can we update to something like this 
onClose={() => setToast(prevState => ({ ...prevState, open: false }))}